app/models/requests/CreateClaimRequest.scala

```diff
@@ -69,7 +69,7 @@ object CreateClaimRequest {
       claimedUnderRegulation <- Some(userAnswers.get(UkRegulationTypePage))
       claimant <- userAnswers.get(ClaimantTypePage)
       claimType <- userAnswers.get(NumberOfEntriesTypePage)
-      noOfEntries <- Some(userAnswers.get(HowManyEntriesPage))
+      noOfEntries <- userAnswers.get(NumberOfEntriesTypePage)
```

Comment:
Can we get the final value here instead of in the yield? i.e userAnswers.get(NumberOfEntriesTypePage).map(_.entries)